Workbooks/Workloads/Collection logs/Collection logs.workbook

```diff
@@ -118,7 +118,7 @@
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "union telegraflogs_CL, waagentlogs_CL\n| order by TimeGenerated desc\n| extend Message = iff(Type == 'telegraflogs_CL', iff('{LogsFilter}' == \"All Logs\", extract(@'.+Z D! \\[outputs.socket_writer\\] (.+)', 1, RawData), \nextract(@'.+Z E! (.+)', 1, RawData)), iff('{LogsFilter}' == \"All Logs\", extract(@'.+Z (.+)', 1, RawData), extract(@'.+Z (.+(Microsoft\\.Azure\\.Monitor\\.WorkloadInsights\\.Test\\.Workload\\.LinuxConfigAgent|Microsoft\\.Azure\\.Monitor\\.WorkloadInsights\\.Test\\.Workload\\.LinuxInstallerAgent)(.+))', 1, RawData) ) )\n| where ('{LogsFilter}' == \"Debug Logs\" and Message != \"\") or ('{LogsFilter}' == \"All Logs\")\n| project Time = TimeGenerated, Message = iff(Message == '', RawData, Message), ['Log type'] = iff(Type == 'telegraflogs_CL', 'Telegraf', 'WA Agent')\n| take 500",
+        "query": "union telegraflogs_CL, waagentlogs_CL\n| order by TimeGenerated desc\n| extend Message = iff(Type == 'telegraflogs_CL', iff('{LogsFilter}' == \"All Logs\", extract(@'.+Z D! \\[outputs.socket_writer\\] (.+)', 1, RawData), \nextract(@'.+Z E! (.+)', 1, RawData)), iff('{LogsFilter}' == \"All Logs\", extract(@'.+Z (.+)', 1, RawData), extract(@'.+Z (.+(Microsoft\\.Azure\\.Monitor\\.WorkloadInsights\\.Test\\.Workload\\.LinuxConfigAgent|Microsoft\\.Azure\\.Monitor\\.WorkloadInsights\\.Test\\.Workload\\.LinuxInstallerAgent)(.+))', 1, RawData) ) )\n| where ('{LogsFilter}' == \"Debug Logs\" and Message != \"\") or ('{LogsFilter}' == \"All Logs\")\n| project Time = TimeGenerated, Message = iff(Message == '', RawData, Message), ['Log type'] = iff(Type == 'telegraflogs_CL', 'Telegraf', iff('{LogsFilter}' == \"Debug Logs\", \"VMExtension\", 'WA Agent'))\n| take 500",

```

Comment:
What is VMExtension in the Log Type column here supposed to mean?

Reply:
We filter the WA Agent logs to show only our "VM extension" logs when user select the Debug logs option. So by default there are two types of logs: Telegraf and Waagent, and filtered logs are named: Telegraf and VMExtension.

Basically to differentiate between the filtered and all logs.

Reply:
This is confusing. Maybe rename VMExtension -> Telegraf setup.

Reply:
fixed